### DISCUSSION TITLE
Error getting transaction information

### DISCUSSION THREAD

**Discussion by xiaomoshi** [njaremko/iota-lib-rs#1247] [General]
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/48044401/189838080-fef5aed9-cd26-4adb-bb27-89bcbc036393.png">
<img width="1488" alt="image" src="https://user-images.githubusercontent.com/48044401/189838173-8997fdd4-402e-4d7b-a959-a8c39b507a2e.png">

**Comment by Thoralf-M** [comment]
@xiaomoshi the nodes only keep a few days of history, everything older gets pruned
The explorer doesn't delete the history, so you can still find it. You can also setup your own node and configure after which time the data gets deleted https://wiki.iota.org/hornet/references/configuration#5-pruning
